packages/design-system-tokens/src/lib/exportCssVars.ts

```diff
@@ -133,15 +132,36 @@ const writeCssFile = (filename: string, importedModule: any, sep: string) => {
  * @returns An exit code based on success writing output
  */
 export const exportCssVars = (fileDescriptors: FileDescriptor[], outPath: string): number => {
-  fileDescriptors.forEach((file) => {
-    const importedModule = require(`${file.moduleImportName}`);
-    // component files do not need a separator
-    const sep = file.baseName.includes('component') ? '' : '-';
-
-    const layoutFilename = `${outPath}/${file.baseName}-layout-tokens.scss`;
-    const cssVarFilename = `${outPath}/${file.baseName}-theme.css`;
-    writeSassLayout(layoutFilename, file, importedModule, sep);
-    writeCssFile(cssVarFilename, importedModule, sep);
+  const groupByTheme = fileDescriptors.reduce((acc: Record<string, any[]>, file) => {
+    const theme = file.baseName.split('-')[0];
+    if (!acc[theme]) {
+      acc[theme] = [];
+    }
+    acc[theme].push(file);
+    return acc;
+  }, {});
+
+  Object.entries(groupByTheme).forEach(([theme, files]) => {
+    let output = '';
+
+    files.forEach((file) => {
+      const importedModule = require(`${file.moduleImportName}`);
+      // Component files do not need a separator
+      let sep = '';
+      output += mapCssVariablesToValues(importedModule, sep);
+
+      // Scss files are only generated from global token files
+      if (!file.baseName.includes('component')) {
```

Comment:
so i tried refactoring this so the scss layout file is written outside the `forEach` loop, but ran into some issues - mainly that the `writeSassLayout()` relies on the `file` parameter to get info to generate the scss vars 😵 

i honestly think that `writeSassLayout()` could probably be simplified since the only scss vars generated come from the theme's global token file and not the component files (right now that func appears to grab ALL tokens, then filters them based on if the token matches the name of a sass var in the `sassFilename` array at the top of the file).

Reply:
in this loop, i only run the write sass func if the `file` isn't a component token file. this ensures it only runs once.

Reply:
i can't wait to get rid of sass